src/components/HeaderBreadcrumbs/HeaderBreadcrumbs.scss

```diff
@@ -6,27 +6,29 @@ $block: '.#{$ns}header-breadcrumbs';
 #{$block} {
     z-index: 11;
 
+    &__list {
+        @include reset-list-style();
+    }
+
     &__item {
+        @include text-size(body-2);
         display: inline-block;
     }
 
     &__text {
-        @include text-size(body-2);
         @include link();
-
         color: var(--g-color-text-secondary);
 
-        &:after {
-            content: '\a0/ ';
-            margin: 0 8px 0 6px;
-            color: var(--g-color-text-secondary);
-        }
-
         &:hover {
             color: var(--g-color-text-primary);
         }
     }
 
+    &__separator {
+        margin: 0 8px 0 6px;
+        color: var(--g-color-text-secondary);
+    }
```

Comment:
lost color for dark theme https://preview.gravity-ui.com/page-constructor/996/?path=/story/blocks-header--breadcrumbs

see in main branch version https://preview.gravity-ui.com/page-constructor/?path=/story/blocks-header--breadcrumbs

Reply:
not fixed